ios/Classes/RoktMethodCallHandler.swift

```diff
@@ -63,28 +83,71 @@ struct RoktMethodCallHandler {
             let callBackId = args["callbackId"] as? Int ?? 0
             var callbackMap = [String: Any] ()
             callbackMap["id"] = callBackId
+            Rokt.events(viewName: viewName, onEvent: { roktEvent in
+                var eventParamMap = [String: String] ()
+                var eventName = ""
+                var placementId: String?
+                if let event = roktEvent as? RoktEvent.FirstPositiveEngagement {
+                    eventName = "FirstPositiveEngagement"
+                    placementId = event.placementId
```

Comment:
Handle the fulfillment here as well?

Reply:
https://github.com/ROKT/rokt-sdk-react-native/pull/132/files#diff-abb31688af2dfba056baf7e09192b94bcd5072727d39c0c4d6abc5c6d6e2100cR102